src/components/datetime/datetime.ts

```diff
@@ -513,7 +513,7 @@ export class DateTime extends Ion implements AfterContentInit, ControlValueAcces
     picker.present(pickerOptions);
 
     this._isOpen = true;
-    picker.onDidDismiss(() => {
+    picker.onDidDismiss((data: any, role: any) => {
```

Comment:
Sorry, I missed your question. I probably added them for debugging and forgot about them. Thanks for reverting the change.